prototypes/prototype-0/docs/verify-service-provider-api.swagger

```diff
@@ -0,0 +1,138 @@
+swagger: "2.0"
+info:
+  description: 'This a proposed API for the Verify Serivce Provider'
+  version: "1.0.0"
+  title: "Verify Service Provider"
+paths:
+  "/authn-request":
+    get:
```

Comment:
a post might be better as it can consume well formed json rather than query params